benchmark/benchmarks.dart

```diff
@@ -4,46 +4,117 @@ import 'package:benchmark_harness/benchmark_harness.dart';
 
 import 'package:r_tree/r_tree.dart';
 
-final int BRANCH_FACTOR = 16;
-
+final int branchFactor = 8;
+final int randomSeed = 3;
 main() {
-  print('Running benchmark...');
+  print('Running benchmarks...');
   var collector = ScoreCollector();
-  InsertBenchmark(collector).report();
+  InsertBenchmark(collector, totalItems: 100).report();
+  InsertBenchmark(collector, totalItems: 1000).report();
+  InsertBenchmark(collector, totalItems: 10000).report();
+
+  LoadBenchmark(collector, totalItems: 100).report();
+  LoadBenchmark(collector, totalItems: 1000).report();
+  LoadBenchmark(collector, totalItems: 10000).report();
```

Comment:
Even load gets slow at this dataset size (around 2 seconds), which is a realistic size seen in production. So this will make it better, but probably not better enough. We'll probably need another mitigation strategy on top of just making insert faster.